### PR TITLE
Yue dev: 纠正了弹性程序的一个错误，加入了GOH-14模型。

### PR DESCRIPTION
**Pull request by ju-liu**

### CODE REVIEW

**Comment by ju-liu** [comment]
@guanjiashen 你看一下这个本构的实现。参考文献你问一下岳新海。

File: src/Model/MaterialModel_GOH14_ST91_Mixed.cpp [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+  const double d2fpsi1 = fk1 * (1.0 + 2.0*fk2*fE1*fE1) * std::exp(fk2*fE1*fE1); 
+  const double d2fpsi2 = fk1 * (1.0 + 2.0*fk2*fE2*fE2) * std::exp(fk2*fE2*fE2); 
+
+  CC.gen_zero();
+
+  CC.add_OutProduct(4.0 * d2fpsi1, PxH1, PxH1);
+  CC.add_OutProduct(4.0 * d2fpsi2, PxH2, PxH2);
+
+  const double val = mpt67 * mu * detFm0d67;
+
```
Comment (guanjiashen) [inline src/Model/MaterialModel_GOH14_ST91_Mixed.cpp:212]: For the isotropic elasticity tensor, I am not sure the result is identical to the one of the NeoHookean model.  A check or a discussion with me is needed.

File: include/MaterialModel_GOH14_ST91_Mixed.hpp [hunk 2]
```diff
@@ -0,0 +1,80 @@
+#ifndef MATERIALMODEL_GOH14_ST91_Mixed_HPP
+#define MATERIALMODEL_GOH14_ST91_Mixed_HPP
+// ==================================================================
+// MaterialModel_GOH14_ST91_Mixed.hpp
+// Quasi-incompressible Gasser-Odgen-Holzapfel model. The model's
+// volumetric part is based on the Simo-Taylor91 penalization. The
+// isochoric isotropic part is the same as the GOH06_Incompressible.
+// The anisotropic part is modified by raplacing the modified right
+// Cauchy-Green tensor by the right Cauchy-Green tensor. This ideal
```
Comment (guanjiashen) [inline include/MaterialModel_GOH14_ST91_Mixed.hpp:9]: There are a few typos: raplacing to replacing, ideal to idea, Auther to Author

**Review by Syj-0525 — APPROVED** [review]

**Comment by ju-liu** [comment]
[GOH06, isochoric anisotropic free-energy term.pdf](https://github.com/APSIS-ANALYSIS/PERIGEE/files/13703081/GOH06.isochoric.anisotropic.free-energy.term.pdf)

**Pull request merged**